.eslintignore

```diff
@@ -3,4 +3,4 @@ __tests__
 node_modules
 coverage
 dist
-src
+src/*.test.js
```

Comment:
No need to ignore it.

Reply:
if I don't ignore this file, running yarn will throw linting error

Reply:
Ok, then can you fix it as well.